### ISSUE TITLE
Potential circularity in the definitions of "I/Me/My" and "My Code", and a typo.

### ISSUE THREAD

**Issue by 11Kilobytes** [copyleft-next/copyleft-next#36]
Quoting the current version of copyleft-next:

```
"I"/"Me"/"My" refers to the individual or legal entity that places My
Work under this License.

"My Code" means the particular work of authorship I license to You
under this License.
```

Firstly "My Work" should be replaced with "My Code" throughout the license. Secondly, "I/Me/My is defined in terms of "My Code," and "My Code" is defined in terms of "I." This circularity is unclear. For clarity, I recommend swapping the definitions of "I/Me/My" and "My Code," and removing the reference to "I" in "My Code."

```
"My Code," refers to any work that is licensed under the terms of this License. 

Each licensee is referred to as "You." 

"I/Me/My" refers to any individual or legal entity that licenses My Code to You.
```

**Comment by ghost** [comment]
I'm no expert, but using the term 'code' appears to arbitrarily limit the variety of authored works that could be licensed under copyleft-next. I don't see why this should be the case, especially if the text goes on to redefine 'My Code' as 'any work that is licensed under the terms of this License'. We've essentially obviated our own distinction.

If the license really does adopt 'code' into its language to fully replace 'work', we end up with additional questions about what _exactly_ does or does not qualify as code within a project. If code broadly refers to anything in a digital medium, we can easily include artwork and other such assets. If not, where is the distinction made?

That aside, I'm all for clarifying the context of the pronouns.

**Comment by keithzg** [comment]
Additionally, "work" has a particular meaning within the realm of copyright law, so I think it's no accident that it's the term being used here.

**Comment by richardfontana** [comment]
These suggestions have been partially implemented in https://github.com/richardfontana/copyleft-next/commit/5e59f165a92e6be61e526f4c7c7e58f87121f100.

@11Kilobytes I have corrected the residual references to 'My Work'. For now I am sticking with 'My Code' as the term for what the GPL calls 'the Program'. I also took the suggestion to take 'I' out of the definition of 'My Code'. I did not fully adopt your suggestions though.

@scionicspectre I understand this point, and if I go back to 'My Work' it will probably be for this reason. Additionally I think it now seems somewhat too 'cutesy', rather as if say a 2005-era Larry Lessig had decided to draft a software license using such a term of art. Yeah, I think I may go back to 'My Work' after all.
